app/models/user.rb

```diff
@@ -7,12 +7,22 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: true
   normalizes :email, with: ->(email) { email.strip.downcase }
 
+  normalizes :first_name, :last_name, with: ->(value) { value.strip.presence }
```

Comment:
By default, the columns have a `nil` value. When the form is saved on the UI (without values), then empty strings are saved in the database.
Generally, I think it's best to normalize `String` values to have sanitized data (i.e. `nil`s instead of empty strings).